tests/sentry/api/endpoints/test_group_hashes.py

```diff
@@ -94,7 +95,7 @@ def test_return_multiple_hashes(self):
     def test_unmerge(self):
         self.login_as(user=self.user)
 
-        group = self.create_group()
+        group = self.create_group(platform="javascript")
```

Comment:
So that I can test for it:

```js
mock_metrics_incr.assert_any_call(
    "grouping.unmerge_issues",
    sample_rate=1.0,
    tags={"platform": "javascript"},
)
```